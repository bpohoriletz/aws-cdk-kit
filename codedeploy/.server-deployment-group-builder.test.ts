import ServerDeploymentGroupBuilder from './server-deployment-group-builder';

describe('new ServerDeploymentGroupBuilder', () => {
  let instance: ServerDeploymentGroupBuilder;

  beforeAll(() => {
    instance = new ServerDeploymentGroupBuilder();
  });

  test('sets defaults', () => {
    const props = instance
      .setApplication()
      .setDeploymentConfig()
      .setName('test-dg')
      .setRole()
      .setInstallAgent()
      .getResult();

    expect(props).toMatchSnapshot();
  });
});
