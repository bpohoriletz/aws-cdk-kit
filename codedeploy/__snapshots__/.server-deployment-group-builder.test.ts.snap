// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new ServerDeploymentGroupBuilder sets defaults 1`] = `
ServerDeploymentGroupProduct {
  "deploymentGroupName": "test-dg",
}
`;
